OpenIPSL/Resources/legal/cla-entity.md

```diff
@@ -0,0 +1,85 @@
+# Open Instance Power System Library - OpenIPSL \ Entity Contributor License Agreement
+
+Thank you for your interest in contributing to Open Instance Power System Library - OpenIPSL  ("We" or "Us").
+
+This contributor agreement ("Agreement") documents the rights granted by contributors to Us. To make this document effective, please sign it and send it to Us by electronic submission, following the instructions [here](https://github.com/cla-assistant/github-action). This is a legally binding document, so please read it carefully before agreeing to it. The Agreement may cover more than one software project managed by Us.
```

Comment:
That link does not give very useful instruction to a normal end user other than a sysadmin that shall implement the CLA on a server. So the submission process should be described for normal human beings.

Reply:
True true, should we link it to a page that has the step-by-step procedure of working with a library in github or should we create our own page and place it in Resources? What do you think is best?

Reply:
Maybe this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) you sent might help me. I will take a look at it first.

Reply:
That document would normally get presented to users providing a PR by the automated CLA process we want to implement. So they ARE already in the middle of accepting the CLA. All they would need as information is what the CLA means not how it is done since this is already triggered.